full-moon/src/tokenizer.rs

```diff
@@ -59,6 +59,8 @@ symbols!(
     Percent => "%",
     // TODO: This only is valid in Roblox
     Pipe => "|",
+    // TODO: This only is valid in Roblox
+    Ampersand => "&",
```

Comment:
Alphabetical order with the rest of this group. You'll notice that this is sometimes out of order, but that's intentionally to make it so things like `.` don't trump `..`. This isn't one of those cases.